docs/developer/editors.md

```diff
@@ -0,0 +1,175 @@
+# Editors
```

Comment:
This is not necessary.

Reply:
does that mean me ## sections should be # and so on?